GeneratorInterface/Pythia8Interface/plugins/EmissionVetoHook1.cc

```diff
@@ -442,7 +442,7 @@ bool EmissionVetoHook1::doVetoFSREmission(int, const Pythia8::Event &e, int iSys
     return false;
 
   // only use for outside resonance vetos in combination with bb4l:FSREmission:veto
```

Comment:
Also the comment line should be updated accordingly

And please fix also the PR description, which is misleading: 
"The showering is set up in such a way that emissions that are outside a resonance, i.e. do not come from top quarks, are handled by the normal EmissionVetoHook1.cc hook, while PowhegHooksBB4L.h handles emissions that are inside a resonance. 
Looking at the single changed line, it can be clearly seen that **the condition needs to be the other way around** for this to happen correctly - if the emission is in resonance, the EmissionVetoHook should not consider it, and leave it up to PowhegHooksBB4L."
(You say that the condition needs to be to other way around, but you repeat the same conditions as before)

Reply:
Hi,
I've updated the PR description to hopefully make it more clear. As far as I can tell, the comment is already correct (it was not describing the code before the change, which is partly how I found the bug).